app/src/main/java/com/codesoom/assignment/auth/dto/AuthenticationRequestDto.java

```diff
@@ -0,0 +1,29 @@
+package com.codesoom.assignment.auth.dto;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotNull;
+
+/**
+ * 사용자 인증 요청 객체
```

Comment:
마침표가 있어야 합니다.
```suggestion
 * 사용자 인증 요청 객체.
```
또는 다음과 같이 수정하면 어떨까요?
```suggestion
 * 사용자 인증 요청.
```

Reply:
객체라는 말은 불필요한 것 같아 아래로 변경했습니다.